### ISSUE TITLE
Error installing datasuit again

### ISSUE THREAD

**Issue by carlitos92jose** [roxas75/rxTools#449]
I just installed the latest commit 3f63c71 and the installation failed issue is back again

**Comment by carlitos92jose** [comment]
Im using a o3ds with 9.2 and 10.6 emunand

**Comment by mariogamer2** [comment]
do you tryed to boot emuNAND? My it work on 10.5 emuNAND.

**Comment by carlitos92jose** [comment]
just tried with the 5ac8aa3 commit and is working normally, the issue is with the latest commit

**Comment by mariogamer2** [comment]
Can you test with b28789724765d5a5bb2f66254a0c6a65a23b3280 ?

**Comment by carlitos92jose** [comment]
@mariogamer2 just tested the b287897 and it worked perfectly, so the issue is definitely the last commit

**Comment by AlbertoSONIC** [comment]
It works for me... Entrypoint?

**Comment by urherenow** [comment]
Just tried with a9lh and it's broken. I could try again via any other method (from a sysnand with 9.0 native_firm firmlaunched...) but I really need to learn git commands. How do I switch from a "checkout" back to normal master or branch?

**Comment by AlbertoSONIC** [comment]
It's broken on firmalunch, you can use arm9loaderhax branch also on other entrypoints like brahma and they'll work fine!

**Comment by carlitos92jose** [comment]
I'm using menuhax as entrypoint

**Comment by urherenow** [comment]
Current build works fine on all entry points, but pasta mode AND installation data suite is broken on a9lh. Works for me through hblauncher_loader+CTRBootManager (on o3ds). after data installation suite is done, I can launch sysnand or emunand via a9lh.

**Comment by mariogamer2** [comment]
Not borcked for me,work perfectly with 10.5 emuNAND and menuhax (rxTools.3dsx renamed to boot.3dsx)

**Comment by carlitos92jose** [comment]
I just installed the latest  012a9c2  release and still the same issue, really weird.

**Comment by urherenow** [comment]
Are you sure you are downloading the proper firm files with the script? Some ISP's have been known to block individual downloads of older native_firm files (Like Americable... see if any of the files in /firm are only 4kb). Using tubehax dns or something similar will cause issues as well.

I'm assuming that you've tried different SD cards as well? Also, are you launching .3dsx or code.bin?

**Comment by carlitos92jose** [comment]
yes, I downloaded the right firm files and if the problem was those firm files, then the older releases of rxtools would not worked for me, I tried launching with both .3dsx and code.bin and the same problem occurred, and I never used tubehax, the problem started when the big code cleanup and the new theme releases on 3f63c71 released

**Comment by urherenow** [comment]
What's the NEWEST one that works for you?

**Comment by carlitos92jose** [comment]
b287897

**Comment by dogtopus** [comment]
Same issue here. Tried 012a9c2 and got error in data installation
BTW the issue only occurs on my O3DS, same version (012a9c2) works fine on my N3DS. Both of my O3DS and N3DS run 10.5 emunand and 9.2 sysnand and both use menuhax + homebrew_launcher + rxTools.3dsx as entry point

**Comment by mariogamer2** [comment]
It's don't hapen for me....

**Comment by dogtopus** [comment]
@mariogamer2 What kind of 3DS are you using? Old 3DS, New 3DS or 2DS?

**Comment by mariogamer2** [comment]
2DS 9.2.0,using menuhax with rxTools as boot.3dsx,with CakeBrah up-to-date (51346773556e7f6bfc23d135e47203d452ab00d1)

**Comment by mariogamer2** [comment]
@urherenow personnally,I learned git by ,myself,googling my error,shearching....

git checkout arm9loaderhax         (go to arm9loaderhax branch)
git checkout master       (master)
